test/wallet-http-test.js

```diff
@@ -85,6 +92,45 @@ for (const witnessOpt of witnessOptions) {
       addr = Address.fromString(str, node.network);
     });
 
+    it('should change passphrase', async () => {
+      await wallet.setPassphrase('initial');
+      await wclient.lock('test');
+
+      // Incorrect Passphrase should not work
+      try {
+        await wallet.unlock('badpass');
+        throw new Error('Unlocked with INCORRECT passphrase!');
+      } catch(e) {
+        assert.strictEqual(e.message, 'Could not decrypt.', e.message);
+      }
+
+      // Correct Passphrase should work
+      try {
+        await wallet.unlock('initial');
+      } catch(e) {
+        throw new Error('Could not unlock with correct passphrase.');
+      }
```

Comment:
If you expect something to work, you shouldn't need to `catch` and then `throw` the error (try it by breaking a test and then running it!) the whole idea behind the bmocha testing framework is that errors are caught by the test runner and reported to the user.